ccpp/data/GFS_typedefs.meta

```diff
@@ -4821,6 +4821,11 @@
   units = count
   dimensions = ()
   type = integer
+[thpsnmp_is_init]
+  standard_name = flag_for_thompson_mp_scheme_initialization
+  long_name = flag carrying scheme initialization status
+  units = flag
```

Comment:
@dustinswales FYI, this variable is missing the required dimensions attribute. I'm currently running RTs for a combined PR for this and Lisa's, so you probably don't need to fix anything since it is already fixed in the release branch. I'm only commenting in case we can't keep the combination for some reason.